src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,155 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
+
+const defaultColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 6,
+  group5: 12,
+};
+
+const mediumColumns = {
+  ...defaultColumns,
+  group3: 5,
+  group4: 5,
+  group5: 10,
+};
+
+const noMargins = {
+  group0: false,
+  group1: false,
+  group2: false,
+  group3: false,
+  group4: false,
+  group5: false,
+};
+
+export const GridWrapperMedium = ({ children }) => {
+  return (
+    <Grid
+      enableGelGutters
+      parentColumns={defaultColumns}
+      columns={defaultColumns}
+    >
+      {children}
+    </Grid>
+  );
+};
+
+GridWrapperMedium.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItem = ({ children, enableMargins }) => {
```

Comment:
I'd suggest calling this GridItemMedium, since it clearly explains that it fits in-between the GridItemSmall and GridItemLarge in how we think about our layouts.